widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/domain/IsSecureEngagementUseCase.kt

```diff
@@ -2,9 +2,13 @@ package com.glia.widgets.core.secureconversations.domain
 
 import com.glia.widgets.chat.ChatType
 import com.glia.widgets.core.engagement.GliaEngagementConfigRepository
+import com.glia.widgets.core.engagement.GliaEngagementRepository
 
-class IsSecureEngagementUseCase(private val engagementConfigRepository: GliaEngagementConfigRepository) {
+class IsSecureEngagementUseCase(private val engagementConfigRepository: GliaEngagementConfigRepository,
+                                private val engagementRepository: GliaEngagementRepository
+) {
     operator fun invoke(): Boolean {
         return engagementConfigRepository.chatType == ChatType.SECURE_MESSAGING
+                && !engagementRepository.hasOngoingEngagement()
     }
 }
```

Comment:
Please add empty line in the end(I know this is legacy)

Reply:
Why? Honestly I've never faced such code style even on Java

Reply:
@DavDo Just to stop showing the warning icon on the Github :)

Reply:
)